website/docs/hardware/assembly-guide/J1-J2-sub-assembly.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: J1-J2-sub-assembly
+title: J1-J2 Sub-Assembly
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
```

Comment:
In general, we should use `require('@site/static/img/...').default` instead of `useBaseUrl`.

See also: https://docusaurus.io/docs/docusaurus-core#useBaseUrl